.github/workflows/validate.yml

```diff
@@ -30,14 +30,17 @@ jobs:
         uses: actions/checkout@v4
       -
         name: Matrix
+        uses: crazy-max/docker-bake-action/subaction/matrix-gen@matrix-gen
```

Comment:
Atm we don't support bake matrix when distributing targets with `jq`:

![image](https://github.com/moby/buildkit/assets/1951866/ca75d948-3819-4242-af83-77962d2aec77)

```json
{
  "group": {
    "default": {
      "targets": [
        "validate"
      ]
    },
    "lint": {
      "targets": [
        "lint-default",
        "lint-labs",
        "lint-nydus",
        "lint-yaml",
        "lint-proto"
      ]
    },
    "validate": {
      "targets": [
        "lint",
        "validate-vendor",
        "validate-doctoc",
        "validate-generated-files",
        "validate-shfmt",
        "validate-docs"
      ]
    }
  },
  ...
```

Testing https://github.com/docker/bake-action/pull/174 so it supported bake matrix feature as well:

![image](https://github.com/moby/buildkit/assets/1951866/001a4bc6-feac-4be1-8a91-04d504d3e518)

Reply:
Actually I can just list keys of `target` property with `jq`.